src/main/java/blackjack/domain/CardGroup.java

```diff
@@ -0,0 +1,35 @@
+package blackjack.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CardGroup {
+
+    private final List<Card> cards = new ArrayList<>();
+
+    public CardGroup(final Card firstCard, final Card secondCard) {
+        cards.add(firstCard);
+        cards.add(secondCard);
+    }
+
+    public int getTotalValue() {
+        return cards.stream().map(Card::getNumber)
+                .mapToInt(CardNumber::getValue)
+                .sum();
```

Comment:
취향마다 차이가 있겠지만, 전반적으로 `.` 마다 개행을 해주는 것은 어떨까요?
(추후에 해당 라인을 삭제할 때도 용이합니다)

```suggestion
        return cards.stream()
                .map(Card::getNumber)
                .mapToInt(CardNumber::getValue)
                .sum();
```